cs3api4lab/auth/reva_password.py

```diff
@@ -0,0 +1,10 @@
+from cs3api4lab.auth.authenticator import Authenticator
+
+
+class RevaPassword(Authenticator):
+
+    def __init__(self, config=None, log=None):
+        super().__init__(config, log)
+
+    def refresh_token(self, client_id=None, client_secret_or_token=None):
```

Comment:
very confusing.. `client_id` as parameter, but then we use the `client_id` from config.. But then this method can be called without anything. I'm not able to follow up.

Reply:
I did remove client_id in public methods for auth, share and file API